sqs_lambda/index.spec.ts

```diff
@@ -161,6 +161,22 @@ describe('SQS Event Handler', () => {
     expect(scope.isDone()).toBeTruthy();
   });
 
+  it('returns success if a NotFoundError is returned from client-api for profile update event', async () => {
```

Comment:
Nice tests! 

I understand the implementation but don't we need to tell FxA to stop pinging for this user somehow? Or do we just expect to get these forever, if there was ever a pocket-FxA user?

Reply:
I'm going with the latter here, but I will raise it with the FxA team today.